src/web/server/htmlTemplate.ts

```diff
@@ -110,32 +110,32 @@ export const htmlTemplate = ({
 	const weAreHiringMessage = `
 <!--
 
-                                    GGGGGGGGG                                   
-                           GGGGGGGGGGGGGGGGGGGGGGGGGG                           
-                       GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG                      
-                    GGGGGGGGGGGGGGGGG      GG   GGGGGGGGGGGGG                   
-                  GGGGGGGGGGGG        GGGGGGGGG      GGGGGGGGGG                 
-                GGGGGGGGGGG         GGGGGGGGGGGGG       GGGGGGGGG               
-              GGGGGGGGGG          GGGGGGGGGGGGGGGGG     GGGGGGGGGGG             
-             GGGGGGGGG           GGGGGGGGGGGGGGGGGGG    GGGGGGGGGGGG            
-            GGGGGGGGG           GGGGGGGGGGGGGGGGGGGGGG  GGGGGGGGGGGGG           
-           GGGGGGGGG            GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG          
-           GGGGGGGG             GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG          
-          GGGGGGGG              GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG         
-          GGGGGGGG              GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG         
-          GGGGGGGG              GGGGGGGGGGGG              GGGGGGGGGGGGG         
-           GGGGGGG              GGGGGGGGGGGGG           GGGGGGGGGGGGGG          
-           GGGGGGGG             GGGGGGGGGGGGG           GGGGGGGGGGGGGG          
-            GGGGGGGG            GGGGGGGGGGGGG           GGGGGGGGGGGGG           
-             GGGGGGGG            GGGGGGGGGGGG           GGGGGGGGGGGG            
-              GGGGGGGGG           GGGGGGGGGGG           GGGGGGGGGGG             
-                GGGGGGGGGG         GGGGGGGGGG           GGGGGGGGG               
-                  GGGGGGGGGGG        GGGGGGGG        GGGGGGGGGG                 
-                    GGGGGGGGGGGGGG      GGGGG  GGGGGGGGGGGGGG                   
-                       GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG                      
-                            GGGGGGGGGGGGGGGGGGGGGGGGG                           
-                                    GGGGGGGGG                                   
-                                              
```

Comment:
Is this the linter or did you get annoyed at empty space?

Reply:
Linter :)